apparmor.d/profiles-a-f/dig

```diff
@@ -24,5 +24,9 @@ profile dig @{exec_path} {
 
   owner @{HOME}/.digrc r,
 
+  /tmp/batch_mode.dig       r,
+  /home/dig/batch_mode.dig  r,
```

Comment:
What is this `/home/dig` directory?

Reply:
A supposed place for files which should only be accessible by dig.

Reply:
I don't get it. `/home/dig` implies there is a `dig` user on the system. Is this your own config. If so, it should go in a `local/dig` file.

Reply:
What about now?